panel/src/panel/content.js

```diff
@@ -156,52 +174,29 @@ export default (panel) => {
 		saveAbortController: null,
 
 		/**
-		 * Updates the values of fields
-		 *
-		 * @param {Object} values
+		 * Updates the form values of the current view
 		 */
-		update(values) {
+		update(values, api = panel.view.props.api) {
 			if (length(values) === 0) {
 				return;
 			}
 
+			if (this.isCurrent(api) === false) {
+				throw new Error("The content in another view cannot be updated");
+			}
+
 			panel.view.props.content = {
 				...panel.view.props.originals,
 				...values
 			};
```

Comment:
`Suggestion:` Continuing my thought from above, wouldn't it make sense to call here `saveLazy(api, panel.view.props.content)`?